rabbitmq/init.sls

```diff
@@ -2,10 +2,10 @@ rabbitmq-server:
   pkgrepo.managed:
     - name: deb http://www.rabbitmq.com/debian/ testing main
     - key_url: http://www.rabbitmq.com/rabbitmq-signing-key-public.asc
+    - require_in:
+      - pkg: rabbitmq-server
   pkg:
-    - installed
-    - require:
-      - pkgrepo: rabbitmq-server
+    - latest
```

Comment:
Should not be needed, it looks like.  Salt refreshes apt when modifying a repo unless you pass refresh_db=False.  https://github.com/saltstack/salt/blob/4e6e19b32113eeb75d45738570223ffb0c6d5a03/salt/modules/aptpkg.py#L1676